ci/run_build.sh

```diff
@@ -50,39 +50,42 @@ popd
 # Smoke tests (actual tests need Docker to run; they don't run within the CI environment)
 for tini in "${BUILD_DIR}/tini" "${BUILD_DIR}/tini-static"; do
   echo "Smoke test for $tini"
-  $tini -h
+  "${tini}" -h
 
   echo "Testing $tini with: true"
-  $tini -vvv true
+  "${tini}" -vvv true
 
   echo "Testing $tini with: false"
-  if $tini -vvv false; then
+  if "${tini}" -vvv false; then
     exit 1
   fi
 
   # Test stdin / stdout are handed over to child
   echo "Testing pipe"
-  echo "exit 0" | $tini -vvv sh
+  echo "exit 0" | "${tini}" -vvv sh
   if [[ ! "$?" -eq "0" ]]; then
     echo "Pipe test failed"
     exit 1
   fi
 
-  # Move files to the dist dir for testing
-  mkdir -p "${DIST_DIR}"
-  cp "${BUILD_DIR}"/tini{,-static,*.rpm,*deb} "${DIST_DIR}"
+  echo "Checking hardening on $tini"
```

Comment:
@ermaker No particular reason (it doesn't make any difference).

I was changing the other references because they were lacking *quoting*, which *is* meaningful and *is* a problem. Since I was changing them anyway, I opted to make them more consistent. I simply missed this one when I introduced it.

Cheers,